wgpu-types/src/lib.rs

```diff
@@ -498,12 +498,17 @@ bitflags::bitflags! {
         /// may also create uniform arrays of storage textures.
         ///
         /// ex.
-        /// - `var textures: array<texture_storage_2d<f32, write>, 10>` (WGSL)
+        /// - `var textures: binding_array<texture_storage_2d<f32, write>, 10>` (WGSL)
         /// - `uniform image2D textures[10]` (GLSL)
         ///
         /// This capability allows them to exist and to be indexed by dynamically uniform
         /// values.
         ///
+        /// Corresponding limits for read + write:
+        /// let mut limits = wgpu::Limits::default();
+        /// limits.max_sampled_textures_per_shader_stage = 64;
```

Comment:
I'm not really sure what these are trying to say - this might imply to the user that they can only use 64 of these at a time.